ktor-client/ktor-client-java/jvm/test/io/ktor/client/engine/java/JavaEngineTests.kt

```diff
@@ -26,29 +27,38 @@ class JavaEngineTests {
 
         val initialNumberOfThreads = Thread.getAllStackTraces().size
         val repeats = 25
+        val executors = ArrayList<ExecutorService>()
```

Comment:
`ArrayList` -> `mutableListOf`?

Reply:
I actually see no point in using this function without arguments. What is the benefit?